### PR TITLE
DOC-9405b: Release notes for 6.6.5

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,10 +5,107 @@
...
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-665]
+== Release 6.6.5 (January 2022)
+
+Couchbase Server 6.6.5 was released in January 2022.
+This maintenance release includes bug fixes in multiple components.
+
+[#fixed-issues-665]
+=== Fixed Issues
```
Comment (b33f) [inline modules/release-notes/pages/relnotes.adoc:17]: Can we mention that Log4J has been update to 2.17.1 please ?

Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:17]: I've added.

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,10 +5,109 @@
...
+==== Data Service
+
+[#table_fixedissues_v665-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34280[MB-34280^]
+| *Summary*: Memcached should disallow DCP connections with names greater than 200 characters
+
+| https://issues.couchbase.com/browse/MB-49321[MB-49321^]
```
Comment (ritamcouchbase) [inline modules/release-notes/pages/relnotes.adoc:49]: Mention of following defect is good - https://issues.couchbase.com/browse/MB-49323

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,10 +5,109 @@
...
+==== Eventing Service
+
+[#table_fixedissues_v665-eventing-service,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-48289[MB-48289^]
+| *Summary*: Fix eventing-producer RSS bloat when eventing-consumers are re-spawned continuously
+
+| https://issues.couchbase.com/browse/MB-48292[MB-48292^]
```
Comment (ritamcouchbase) [inline modules/release-notes/pages/relnotes.adoc]: This one can be removed.

**Review by ritamcouchbase — COMMENTED** [review]
Left few comments.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc:49]: I have added.

File: modules/release-notes/pages/relnotes.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I have removed.

**Review by arod1987 — APPROVED** [review]

**Review by RichardSmedley — APPROVED** [review]
LGTM

**Pull request merged**